### ISSUE TITLE
Dropdown only printing first option again

### ISSUE THREAD

**Issue by JEKO10** [MatthewHerbst/react-to-print#751]
The ``` <select> ``` tag only renders the first option on print, regardless of whether the user has chosen the option. 

I noticed that a bug has been fixed, but it is still present in version ^3.0.1.

**Comment by siaikin** [comment]
This is the default behavior of `<select>`, and you can add a blank 'default option' to make it look blank.

Reference link: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select

```html
<select name="pets" id="pet-select">

  <option value=""></option> // Empty option
  <option value="goldfish">Goldfish</option>
</select>
```

**Comment by JEKO10** [comment]
That way it will always print a blank option... Anyway, I found a solution: 

You need to have the ```selected``` attribute on the ```<option>``` that you've selected to trigger innerHTML to change. In my solution, I remove the  ```selected``` attribute from all options before choosing another one to avoid having multiple ```selected```. Then, I set the ```selected``` attribute only for the last one clicked.

The ```document.querySelectorAll("option");``` retrieves all options on a page, while 

```document.querySelectorAll("option:checked")``` only selects options that are currently selected.

```
 useEffect(() => {
    const options = document.querySelectorAll("option");
    options.forEach((option) => option.removeAttribute("selected"));

    for (const option of document.querySelectorAll("option:checked")) {
      option.setAttribute("selected", "");
    }
  }, [state value that is changed in select onChange event]);
  ```
  
Hey @MatthewHerbst, could I contribute to your amazing library if this solution is acceptable for the issue? :)

**Comment by siaikin** [comment]
@MatthewHerbst M This looks like a regression issue, I found the missing code in the commit record.

https://github.com/MatthewHerbst/react-to-print/blob/977b034c10932f91eca13468c2042dc577fcf73d/src/index.tsx#L427

**Comment by MatthewHerbst** [comment]
Hey, I'm traveling right now but should be able to take a look in a few hours. Apologies if there is a regression. And contributions are always appreciated!

**Comment by MatthewHerbst** [comment]
Ah, so I had removed these code blocks thinking they weren't needed anymore. Seems that's true of the input, radio, and checkboxes, but this is still needed for selects. Fix shipping now

**Comment by MatthewHerbst** [comment]
Fix published in `v3.0.2`, please let me know if you run into any other issues. Thanks for reporting this!
